include/openssl/ec.h

```diff
@@ -392,15 +392,31 @@ OPENSSL_EXPORT int EC_GROUP_set_generator(EC_GROUP *group,
                                           const BIGNUM *cofactor);
 
 
-// EC_POINT_point2bn converts an |EC_POINT| to a |BIGNUM| by serializing the
-// point into the X9.62 form given by |form| then interpretting it as a BIGNUM.
+// EC_POINT_point2bn serialises |point| into the X9.62 form given by |form|
```

Comment:
what is this format exactly? like if the point is `(x, y)` what should the serialized output be?